packages/dnb-eufemia/src/shared/locales/en-GB.js

```diff
@@ -130,11 +130,14 @@ export default {
     Upload: {
       title: 'Upload documents',
       text: 'Drag & drop your files or choose files to upload.',
+      textSingleFileLimit:
```

Comment:
```suggestion
      textSingular:
```

Should probably be enough to understand the purpose.

Reply:
I'll change from `textSingleFileLimit` to `textSingular`, and from `buttonTextSingleFileLimit` to `buttonTextSingular` 👍